google-cloud-spanner/src/main/java/com/google/cloud/spanner/InstanceInfo.java

```diff
@@ -35,13 +36,16 @@ public enum InstanceField implements FieldSelector {
     DISPLAY_NAME("display_name"),
     NODE_COUNT("node_count"),
     PROCESSING_UNITS("processing_units"),
+    AUTOSCALING_CONFIG("autoscaling_config"),
     LABELS("labels");
 
     static InstanceField[] defaultFieldsToUpdate(InstanceInfo info) {
-      if (info.getNodeCount() > 0) {
-        return new InstanceField[] {DISPLAY_NAME, NODE_COUNT, LABELS};
+      if (info.getAutoscalingConfig() != null) {
+        return new InstanceField[] {DISPLAY_NAME, AUTOSCALING_CONFIG, LABELS};
+      } else if (info.getNodeCount() > 0) {
+        return new InstanceField[] {DISPLAY_NAME, AUTOSCALING_CONFIG, NODE_COUNT, LABELS};
       } else {
-        return new InstanceField[] {DISPLAY_NAME, PROCESSING_UNITS, LABELS};
+        return new InstanceField[] {DISPLAY_NAME, AUTOSCALING_CONFIG, PROCESSING_UNITS, LABELS};
```

Comment:
Is it expected to contain both `AUTOSCALING_CONFIG` and  `PROCESSING_UNITS` ?

Reply:
Same above.